html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li>
+        <a href="/html-relative-links/index.html">Home Page</a>
+      </li>
+      <li>
+        <a href="about-me.html">About me</a>
+      </li>
+      <li>
+        <a href="/html-relative-links/projects/projects.html">Projects</a>
+      </li>
+      <li>
+        <a href="/html-relative-links/contact-us/contact-us.html">Contact Us</a>
+      </li>
+    </ul>
+    <p>Hello About Me!</p>
+    <img
+      src="/html-relative-links/images/c.png"
```

Comment:
Instead of starting at `/html-relative-links`, you should be using `../` to move up directories. You want to take the shortest distance possible